catalog/forms.py

```diff
@@ -1,7 +1,44 @@
 from django import forms
-from .models import Product, Category
+from .models import Product
+from django.core.exceptions import ValidationError
+
+FORBIDDEN_WORDS = ['казино', 'криптовалюта', 'крипта', 'биржа', 'дешево', 'бесплатно', 'обман', 'полиция', 'радар']
+
 
 class ProductForm(forms.ModelForm):
     class Meta:
         model = Product
-        fields = ['name', 'description', 'image', 'category', 'price']
+        fields = ['name', 'description', 'price', 'image', 'category']
+
+    def clean_name(self):
+        name = self.cleaned_data.get('name')

```

Comment:
Поля `name`, `description` и `price` обязательные, поэтому нужно получать так
```suggestion
        name = self.cleaned_data['name']

```
Метод `.get` подразумевает получение None в случае отсутствия ключа, а значит проверка ниже `name.lower()` упадет с ошибкой, ведь метода `lower` у None нет